src/compiler/emit.jl

```diff
@@ -95,6 +95,7 @@ end
 
 varargs(m::Method, n) = m.isva ? n - m.nargs + 1 : nothing
 
+# returns IR code for backward pass, or nothing
```

Comment:
Would be good to explain under what conditions nothing is returned.

Reply:
If only I knew...I _think_ what is meant to happen is that metadata for non-existing method is `nothing`, and that means that the IR backward pass can not be generated?